tests/foreman/api/test_docker.py

```diff
@@ -828,12 +755,11 @@ def test_promote_multiple_docker_repo_content_view(self):
         cvv = content_view.read().version[0]
         self.assertEqual(len(cvv.read().environment), 1)
 
-        for i in range(1, randint(2, 5)):
```

Comment:
`range` is a little bit tricky. FYI, `range(1, 2)` will return `[1]`, so returning back as it was is not OK. same with `range(2, 3)`.
I guess `for i in range(1, randint(3, 6))` should work instead:
```python
>>> range(1, 3)
[1, 2]
```
So i guess we shall decide, what's less ugly - `for i in range(1, randint(3, 6))` or `i+2` :smile: